3.Tests/UI/StepDefinitions/User/LogoutSteps.cs

```diff
@@ -0,0 +1,44 @@
+﻿using OpenQA.Selenium;
+using SeleniumTest.Core.Drivers;
+using SeleniumTest.Core.Interfaces;
+using SeleniumTest.Tests.Steps.Commons;
+using TechTalk.SpecFlow;
+using Views.WebAppPages;
+
+namespace SeleniumTest.Tests.Steps.User;
+
+[Binding]
+[Scope(Feature = "User Logout")]
+public class LogoutStepDefinitions : CommonSteps
+{
+    private readonly HomePage _homePage;
+    private readonly IGenericWebDriver _driver;
+    private readonly ScenarioContext _scenarioContext;
+
+    public LogoutStepDefinitions(ScenarioContext scenarioContext, ChromeWebDriver driver)
+        : base(scenarioContext, driver)
+    {
+        _scenarioContext = scenarioContext;
+        _driver = driver;
+        _homePage = new HomePage(_driver);
+    }
+
+    [TearDown]
```

Comment:
Avoid using NUnit, we should only rely on Specflow, so you should move this code to an AfterScenario or AfterTestRun hook